vyxal/elements.py

```diff
@@ -3908,7 +3923,7 @@ def vy_print(lhs, end="\n", ctx=None):
         res = lhs(ctx.stacks[-1], lhs, ctx=ctx)[-1]
         vy_print(res, ctx=ctx)
     else:
-        if ts == NUMBER_TYPE:
+        if is_sympy(lhs):
             if ctx.print_decimals:
                 lhs = eval(sympy.pycode(sympy.nsimplify(lhs)))
```

Comment:
This looks unnecessarily complicated. Can't `sympy.N` be used here?

Reply:
The problem with that is that it's fixed length and can have too many leading 0s sometimes. This way allows python to determine precision.